src/system/SystemClock.h

```diff
@@ -184,6 +184,10 @@ class ClockBase
      * @returns             Elapsed time in milliseconds since an arbitrary, platform-defined epoch.
      */
     virtual Milliseconds64 GetMonotonicMilliseconds64() = 0;
+
+    virtual CHIP_ERROR GetClock_RealTime(uint64_t & curTime)   = 0;
```

Comment:
Can these be made type-safe? Either way, please document what these values are.

Reply:
just update these value.

Reply:
I think i can make it type-safe via applying Clock::Milliseconds64